Mlem/App/Views/Pages/Community/CommunityView.swift

```diff
@@ -95,12 +95,18 @@ struct CommunityView: View {
             .environment(\.communityContext, community)
         }
         .background(postSize.tiled ? palette.groupedBackground : palette.background)
-        .loadFeed(postFeedLoader)
+        .toolbar {
+            ToolbarItemGroup(placement: .secondaryAction) {
+                ForEach(community.menuActions(navigation: navigation), id: \.id) { action in
+                    MenuButton(action: action)
+                }
```

Comment:
*nit* avoid calling `menuActions` until needed

```suggestion
                MenuButtons { community.menuActions(navigation: navigation) }
```